### DISCUSSION TITLE
How do indicate the code chunk language?

### DISCUSSION THREAD

**Discussion by jamespeapen** [quarto-dev/quarto-cli#3768] [Q&A] [answered]
I have a document with code chunks in different languages and I'd like each code chunk to have an indicator of its language.
Is there an options for this or a way to do this with css?

**Comment by camilogarciabotero** [comment]
Do you mean something like code-block-language-label?

Here is a previous discussion: https://github.com/quarto-dev/quarto-cli/discussions/623

**Reply by camilogarciabotero** [reply]
It would be awesome if it gets implemented!

**Comment by jamespeapen** [comment, marked as answer]
See https://github.com/quarto-dev/quarto-cli/discussions/623#discussioncomment-7922937. Marking as resolved here.
